src/java/org/apache/cassandra/config/YamlConfigurationLoader.java

```diff
@@ -137,9 +142,48 @@ public Config loadConfig(URL url) throws ConfigurationException
         }
         catch (YAMLException e)
         {
-            throw new ConfigurationException("Invalid yaml: " + url + SystemUtils.LINE_SEPARATOR
-                                             +  " Error: " + e.getMessage(), false);
+            throw new ConfigurationException("Invalid yaml: " + url, e);
```

Comment:
I don't see how this change was intentional, I will return it pre-commit

Reply:
Actually we absolutely need this change, otherwise the users won't see the exceptions for units, etc